### PR TITLE
Will's Auth Challenge

### PR DESCRIPTION
**Pull request by Willcowen**

### CODE REVIEW

File: src/client/App.jsx [hunk 1]
```diff
@@ -11,80 +11,83 @@ function App() {
...
+
   const handleCreateMovie = async ({ title, description, runtimeMins }) => {
 
     const options = {
       method: 'POST',
       headers: {
-          'content-type': 'application/json',
-          'authorization': 'Bearer ' + localStorage.getItem('jwt')
+        'content-type': 'application/json',
+        'authorization': 'Bearer ' + localStorage.getItem('jwt')
```
Comment (therearedoors) [inline src/client/App.jsx:87]: noticed this the other day - what does 'Bearer' achieve in this string? Mine seemed to work with just the token.

File: src/client/App.jsx [hunk 2]
```diff
@@ -15,15 +15,76 @@ function App() {
...
+         title: title,
+         description: description,
+         runtimeMins: runtimeMins
+      }),
+  }
+
+  fetch('http://localhost:4000/movie', options)
+  .then(res => {
+    res.json().then(json => {
+      if(res.ok) {
```
Comment (therearedoors) [inline src/client/App.jsx]: I didn't know about Response.ok, just looked it up: https://developer.mozilla.org/en-US/docs/Web/API/Response/ok, this is quite handy!

**Review by alsyoufomar — COMMENTED** [review]
what a nice neat code, nothing to add actually, well done Will!

File: src/client/App.jsx (hunk 1 above)
Comment (Willcowen) [reply src/client/App.jsx:87]: After a quick read online, it seems there are different types of authorisation. "Many web servers support multiple methods of authorization. In those cases sending just the token isn't sufficient." - taken from stackexchange.

**Pull request closed without merging**